doc8/parser.py

```diff
@@ -80,17 +80,20 @@ def _read(self):
         with self._read_lock:
             if not self._has_read:
                 with open(self.filename, "rb") as fh:
```

Comment:
Why we are opening text files as binary? Opening as text should allow us to use universal newlines support in python.

Reply:
I'm sure Markus will correct me if I am wrong, but wouldn't that disable D004 (Literal carriage return)?

Reply:
Yes, otherwise you will lose the original line endings (they are converted to `\n`) and D004 becomes obsolete. Would be fine for those who have filed the issue, but probably not for you?